site/app/views/grading/ElectronicGraderView.php

```diff
@@ -701,9 +701,18 @@ public function renderRubricPanel(Gradeable $gradeable, User $user) {
      * @return string
      */
     public function renderRegradePanel(Gradeable $gradeable) {
-        return $this->core->getOutput()->renderTwigTemplate("grading/electronic/RegradePanel.twig", [
-            "gradeable" => $gradeable
-        ]);
+        $return = <<<HTML
+<div id="regrade_info" class = "draggable rubric_panel" style="right: 15px; bottom: 40px;width: 48%; height: 30%">
```

Comment:
Why are you putting HEREDOC/html in php when there is a perfectly good twig file to edit? The whole point of the twig refactor was to _not_ have html directly in php.